go.mod

```diff
@@ -6,6 +6,7 @@ go 1.21
 replace github.com/rancher/wrangler v1.1.1 => github.com/rancher/wrangler v1.1.1-0.20230831050635-df1bd5aae9df
 
 replace (
+	github.com/rancher/rancher/pkg/apis => github.com/pmatseykanets/rancher/pkg/apis v0.0.0-20240123143352-5df802d576a6
```

Comment:
Is this temporary or just something that never got removed?

Reply:
It's temporary https://github.com/rancher/webhook/pull/327#issuecomment-1912332010

Reply:
yes, temporary